Secure API Endpoints with JWT Access Token (#169)
ral-facilities/inventory-management-system-api: README.md

Ensure that Docker is installed and running on your machine before proceeding.

1. Create a `.env` file alongside the `.env.example` file. Use the example file as a reference and modify the values accordingly.
    ```bash
    cp .env.example .env
    ```

2. Create a `logging.ini` file alongside the `logging.example.ini` file. Use the example file as a reference and modify it accordingly:
    ```bash
    cp logging.example.ini logging.ini
    ```

> 3. (**Required only if JWT Auth is enabled**) Inside the `keys` directory at the root of the project directory, create a copy of the public key generated by the authentication component. This is needed for decoding of JWT access tokens signed by the corresponding private key.

Reviewer: Could probably mention as well to ensure there is a `.keep` file, otherwise don't people risk just losing their public key as it is in the `.gitignore` file

Reviewer 2: I don't think this is an issue, the .keep just ensures the folder is committed if git thinks its empty, which it will do since the jwt is in the git ignore. We don't want anyone committing their jwt public key accidentally.

> 3. (**Required only if JWT Auth is enabled**) Inside the `keys` directory at the root of the project directory, create a copy of the public key generated by the authentication component. This is needed for decoding of JWT access tokens signed by the corresponding private key.

Reviewer 2: This should mention creating the keys directory like ldap-jwt-auth as it is no longer committed.

Author: Sorry I forgot to revert the changes for this. Addressed now in https://github.com/ral-facilities/inventory-management-system-api/pull/169/commits/98866f6433220d558752f7356775ae96515d920c.

> 3. (**Required only if JWT Auth is enabled**) Create a `keys` directory in the root of the project directory and inside it create a copy of the public key generated by the authentication component. This is needed for decoding of JWT access tokens signed by the corresponding private key.

Author: `keys` dir has now been committed so there is no need to be created.
```suggestion
3. (**Required only if JWT Auth is enabled**) Inside the `keys` directory at the root of the project directory, create a copy of the public key generated by the authentication component. This is needed for decoding of JWT access tokens signed by the corresponding private key.
```
